package.json

```diff
@@ -32,15 +30,13 @@
   },
   "devDependencies": {
     "@types/jest": "^29.4.0",
-    "@types/node": "^18.14.1",
-    "axios": "^1.3.4",
-    "cucumber": "^6.0.5",
+    "@types/node": "^18.13.0",
+    "axios": "^1.3.2",
     "ergol": "^1.0.2",
     "eslint-plugin-kuzzle": "^0.0.6",
-    "jest": "^29.4.3",
-    "kuzzle": "^2.21.0",
-    "kuzzle-sdk": "^7.10.6",
-    "should": "^13.2.3",
+    "jest": "^29.4.1",
+    "kuzzle": "^2.20.2",
+    "kuzzle-sdk": "^7.10.5",
```

Comment:
Why change versions here ?

Reply:
I bumped to the latest version mainly because I needed the latest sdk version